### ISSUE TITLE
Default value for JQuery.detach()

### ISSUE THREAD

**Issue by essentin** [UdashFramework/udash-core#174]
Wrapper for `detach()` at `io.udash.wrappers.jquery.JQuery` does not have default `js.native` value, which is a bummer because to use it without parameters you have to go with something like `scala.scalajs.js.Dynamic.global.jQuery(...).detach()`

**Comment by Starzu** [comment]
Fixed in the wrapper's repo: https://github.com/UdashFramework/scala-js-jquery/pull/14
